prinz/src/main/scala/pl/touk/nussknacker/prinz/model/proxy/ProxiedModelComposedInputParam.scala

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.prinz.model.proxy
+
+import pl.touk.nussknacker.prinz.model.proxy.ProxiedModelComposedInputParam.{ComposedParamsSupplier, ParamsExtractor}
+import pl.touk.nussknacker.prinz.model.{ModelMetadata, SignatureName}
+
+import scala.concurrent.Future
+
+final case class ProxiedModelComposedInputParam[T](paramsSupplier: ComposedParamsSupplier[T],
```

Comment:
1. should be Composite instead of Composed?
2. How do we know which parameters will be supplied? There has to be some way to detect that?  E.g. method: 
    Model(Signature) => F?[List[SignatureName]] - which will tell which parameters will be used? 
2. Wouldn't it be easier to have trait with one method for parameter supply/extraction?

Reply:
Added proxy.transofrmer which realizes given task in described way